daemon/emer-aggregate-timer-impl.c

```diff
@@ -114,7 +114,8 @@ emer_aggregate_timer_impl_new (EmerAggregateTally *tally,
   if (payload)
     g_variant_take_ref (payload);
 
-  cache_key = g_variant_new ("(u@ayvmv)",
+  cache_key = g_variant_new ("(su@ayvmv)",
+                             sender_name,
                              unix_user_id,
```

Comment:
This is fine, but actually you don't need both the sender_name and unix_user_id in this key, since there is exactly 1 Unix user ID for a D-Bus unique name. (He says, shortly before wondering what happens if you connect to the bus as root, fork, and change your effective user id...)